finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
Good point. Better to do a `get` in a `try..catch`, and do the create in the `catch`:
```python
try:
    money = models.IndependentMoney.objects.get(source='NF', source_xact_id= source_xact_id)
except models.IndependentMoney.DoesNotExist:
    money = models.IndependentMoney(source='NF', source_xact_id= source_xact_id, ...)
    money.save()
```

That's pseudo-code...